pkg/collector/corechecks/sbom/processor.go

```diff
@@ -380,12 +394,33 @@ func (p *processor) processImageSBOM(img *workloadmeta.ContainerImageMetadata) {
 		}
 
 		if len(repoDigests) == 0 {
-			log.Infof("The image %s has no repo digest for repo %s", img.ID, repo)
-			continue
+			allowMissingRepodigest := p.cfg.GetBool("sbom.container_image.allow_missing_repodigest")
+			if !allowMissingRepodigest || len(img.RepoDigests) != 0 {
+				log.Infof("The image %s has no repo digest for repo %s, skipping", img.ID, repo)
+				continue
+			}
+
+			// locally built images have no repo digest so we generate one in the
+			// form of local/my-image:my-tag@sha256:xxxxx
+			repoDigest := "local/" + img.Name + "@" + img.ID
```

Comment:
Just to remove any confusion: does `img.Name` contain both the image name and tag here?

Reply:
Yes